Droidcon-Boston/app/src/main/AndroidManifest.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    package="com.mentalmachines.droidcon_boston">
+    package="com.mentalmachines.droidconboston">
```

Comment:
does this break upgradability for older users?  Although I'm not sure why they'd have the last year's app still installed :-)

Reply:
I thought about this too, but I wasn't sure if it ws the package name in the manifest, or the applicationId in build.gradle that really mattered (which is unchanged on this branch).